moped-editor/src/queries/workgroups.js

```diff
@@ -2,7 +2,10 @@ import { gql } from "@apollo/client";
 
 export const WORKGROUPS_QUERY = gql`
   query GetWorkgroups {
-    moped_workgroup(order_by: {workgroup_name: asc}) {
+    moped_workgroup(
+      order_by: { workgroup_name: asc }
+      where: { is_deleted: { _eq: false } }
+    ) {
```

Comment:
looks great! i did a quick scan of instances of `moped_workgroup` in the editor and couldn't find anything else that would be affected by this 👍

Reply:
thanks for checking! 🙏